content/docs/studio/user-guide/model-registry/what-is-a-model-registry.md

```diff
@@ -20,9 +20,9 @@ open-source Git-based tools [GTO] and [MLEM].
   be in S3, GCP, or any other of your remote, or local, storages.
 
 The model registry provides an interactive web interface to the metadata files
-and Git tags. You can also use the model registry through the [GTO CLI and
-Python API][gto]. Any updates that you make from the CLI or API are also
-reflected in the model registry in Iterative Studio.
+and Git tags. You can also use the model registry through the [`gto` CLI]. Any
```

Comment:
There's no GTO Python API right @aguschin ? Thanks

Reply:
No docs for it. The API still exists, although can be removed in future.